pkgs/development/libraries/dlib/default.nix

```diff
@@ -1,5 +1,8 @@
 { stdenv, lib, fetchFromGitHub, cmake, pkgconfig, libpng, libjpeg
 , guiSupport ? false, libX11
+
+  # see http://dlib.net/compile.html
+, avxInstructions ? true
```

Comment:
`avxSupport` to be in line with other derivations

Reply:
you're right, fixed.